includes/managers/icons.php

```diff
@@ -280,14 +280,14 @@ public static function get_font_icon_svg( $icon, $attributes = [] ) {
 
 		$attributes['class'][] = self::FONT_ICON_SVG_CLASS_NAME;
 		$attributes['class'][] = 'e-' . $icon_data['key'];
+		$attributes['viewBox'] = '0 0 ' . $icon_data['width'] . ' ' . $icon_data['height'];
 
 		/**
 		 * If in edit mode inline the full svg, otherwise use the symbol.
 		 * Will be displayed only after page update or widget "blur".
 		 */
 		if ( Plugin::$instance->editor->is_edit_mode() ) {
-			return '<svg xmlns="http://www.w3.org/2000/svg" ' . Utils::render_html_attributes( $attributes ) . '
-				viewBox="0 0 ' . esc_attr( $icon_data['width'] ) . ' ' . esc_attr( $icon_data['height'] ) . '">
+			return '<svg xmlns="http://www.w3.org/2000/svg" ' . Utils::render_html_attributes( $attributes ) . '">
```

Comment:
@matipojo 
Open the lines below, see the `svg` + `use` prints

Reply:
Can you attach screenshots before/after for several icons?

Reply:
Done